i18n/push_translations.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis CI"
+
+head_ref=$(git rev-parse HEAD)
```

Comment:
Some comments explaining the `if` cases would be nice.

Reply:
I couldn't really add more than in the `echo` below (I only copied this from some tutorial).